File: CargoFlash.ROMS.Model/Master/AccountGroup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace CargoFlash.Cargo.Model.Master
{
    #region Account Group Description
    /*
	*****************************************************************************
	Class Name:		AccountGroup   
	Purpose:		This class used to handle
	Company:		CargoFlash Infotech Pvt Ltd.
	Author:			Tarun Kumar
	Created On:		21 May 2014
    Updated By:    
	Updated On:
	Approved By:    
	Approved On:	
	*****************************************************************************
	*/
    #endregion

    [KnownType(typeof(AccountGroup))]
    public class AccountGroup
    {
        public int SNo { get; set; }
        public String Name { get; set; }
        public bool IsActive { get; set; }
        public string CreatedBy { get; set; }
        public string UpdatedBy { get; set; }
        public String Active { get; set; }
    }


    [KnownType(typeof(AccountGroupTrans))]
    public class AccountGroupTrans
    {
        public int SNo { get; set; }
        public int AccountGroupSNo { get; set; }
        public String AccountSNo { get; set; }
        public int IsActive { get; set; }
        public string CreatedBy { get; set; }
        public string UpdatedBy { get; set; }
        public String Active { get; set; }
        public String HdnAccountSNo { get; set; }
    }
}

File: CargoFlashCargoWebApps/JScript/Master/AccountGroup.js
/*
*****************************************************************************
Javascript Name:	AccountGroupJS     
Purpose:		    This JS used to get Grid data for Account Group Trans.
Company:		    CargoFlash Infotech Pvt Ltd.
Author:			    Tarun Kumar
Created On:		    21 May 2014
Updated By:         
Updated On:	        
Approved By:        
Approved On:	    
*****************************************************************************
*/


$(document).ready(function () {
    $('#Name').keypress(function (e) {

        if (e.keyCode != 32)
            return true;
        else
            return false;
    })

    $(document).on("contextmenu", function (e) {
        alert('Right click disabled');
        return false;
    });

    $(document).on('drop', function () {
        return false;
    });
});




$(function () {
    cfi.ValidateForm();
    $('#aspnetForm').attr("enctype", "multipart/form-data");
    var tabStrip1 = $("#ApplicationTabs").kendoTabStrip().data("kendoTabStrip");
});
var dbTableName = 'AccountGroupTrans';
function AccountGroupTransGrid() {
    if (getQueryStringValue("FormAction").toUpperCase() == "NEW" || getQueryStringValue("FormAction").toUpperCase() == "DUPLICATE") {
        ShowMessage('info', 'Need your Kind Attention!', "Account Group Trans Details can be added in Edit/Update mode only.");
        return;
    }
    else {
        var pageType = $('#hdnPageType').val();
        cfi.ValidateForm();
        $('#tbl' + dbTableName).appendGrid({
            tableID: 'tbl' + dbTableName,
            contentEditable: pageType != 'View',
            isGetRecord: true,
            tableColumns: 'SNo,Name,IsActive,CreatedBy,UpdatedBy',
            masterTableSNo: $('#hdnAccountGroupSNo').val(),
            currentPage: 1, itemsPerPage: 5, whereCondition: null, sort: '',
            servicePath: './Services/Master/AccountGroupService.svc',
            getRecordServiceMethod: 'Get' + dbTableName + 'Record',
            createUpdateServiceMethod: 'createUpdate' + dbTableName,
            deleteServiceMethod: 'delete' + dbTableName,
            caption: 'Account Group Trans',
            initRows: 1,
            columns: [
                { name: 'SNo', type: 'hidden', value: 0 },
                { name: 'AccountGroupSNo', type: 'hidden', value: $('#hdnAccountGroupSNo').val() },
                { name: 'AccountSNo', display: 'Account Name', type: 'text', ctrlAttr: { maxlength: 100, controltype: 'autocomplete' }, ctrlCss: { width: '140px', height: '20px' }, isRequired: true, AutoCompleteName: 'Accountgroup_AccountSNo', filterField: 'Name' },
                { name: pageType == 'Edit' ? 'IsActive' : 'Active', display: 'Active', type: 'radiolist', ctrlOptions: { 0: 'No', 1: 'Yes' }, selectedIndex: 1, onClick: function (evt, rowIndex) { } },
                { name: 'CreatedBy', type: 'hidden', value: $('#hdnCreatedBy').val() },
                { name: 'UpdatedBy', type: 'hidden', value: $('#hdnUpdatedBy').val() }
            ],
            isPaging: true,
        });     
    }
}